content_sync/models.py

```diff
@@ -10,7 +10,7 @@ class ContentSyncState(TimestampedModel):
 
     content = models.OneToOneField(
         WebsiteContent,
-        on_delete=models.PROTECT,
+        on_delete=models.SET_NULL,
```

Comment:
In a shell I noticed that after deleting a `WebsiteContent` object, retrieving its `ContentSyncState` would throw an exception:

```
ContentSyncState.objects.order_by("created_on").last()
>AttributeError: 'NoneType' object has no attribute 'title'
```

Can be avoided by modifying the `__str__` method to something like this:
```
return f"Sync State for content: {self.content.title if self.content else None}"
```